### DISCUSSION TITLE
How to add <link> tag to <head> ?

### DISCUSSION THREAD

**Discussion by regul8or** [reflex-dev/reflex#3117] [Q&A] [answered]
Hello. There was an issue #662 (Possibly to add link tags in <head>). It's closed as completed. But I can't find how to do it. Every bit of information I find is about `rx.script`. Is there a way to do it?

**Comment by masenf** [comment, marked as answer]
You can only do it for the whole app, but it looks something like this

```python
app = rx.App(
    head_components=[
        rx.el.link(
            href="fancy.css",
            rel="alternate stylesheet",
            title="Fancy",
        ),
    ]
)
```

You can also add meta tags via the same method with `rx.el.meta`.

And it's not really supported, but you can also include raw inline styles via

```python
rx.el.element.Element.create("p {color: blue}", tag="style"),
```

**Reply by regul8or** [reply]
Great, thank you!
